doc/source/api.rst

```diff
@@ -52,11 +54,13 @@ Image
     Image
     Image.convert_to_CAS_column
     Image.convert_wide_to_numpy
+    Image.convert_numpyt_to_wide
```

Comment:
I think the "t" should be removed after "numpy" here

Reply:
Good catch. Thanks! That should be fixed, now.